starter_code/index.js

```diff
@@ -1 +1,26 @@
-const Elevator = require('./elevator.js');
+const Elevator = require("./elevator.js");
+const Person = require("./person.js");
+
+var elevator = new Elevator();
+
+elevator.floorUp();
+elevator.floorUp();
+
+elevator.update();
+
+var person1 = new Person("John", 2, 3);
+var person3 = new Person("Maria", 2, 4);
+var person2 = new Person("Peter", 0, 6);
+
+elevator.call(person1);
+elevator.call(person2);
+elevator.call(person3);
+
+elevator._passengersEnter();
```

Comment:
same for this method. Should be called behind-the-scences in your logic, not directly by the application